data/mods/innawood/npcs/missiondef.json

```diff
@@ -0,0 +1,421 @@
+[
+  {
+    "id": "MISSION_GET_INHALER",
+    "type": "mission_definition",
+    "name": { "str": "Find Inhaler Rock" },
```

Comment:
Spurge flower or tea would fit the same anti-asthmatic need

Reply:
I agree that spurge tea would be a much better mission goal here.

Reply:
Done